app/views/guide_typography.html

```diff
@@ -133,9 +133,11 @@ <h3 class="heading-medium" id="typography-links">Links</h3>
 </pre>
 
   <h3 class="heading-medium" id="typography-lists">Lists</h3>
-  <p class="text">
-    List items start with a lowercase letter and have no full stop at the end.
-  </p>
+  
+  <ul class="list list-bullet text">
+    <li>list items start with a lowercase letter and have no full stop at the end</li> 
```

Comment:
Sure, So leave my snippet changes as they are, but replace the stuff in the main view with a link to the style guide?